src/scss/components/_simple-select-list.scss

```diff
@@ -24,6 +26,43 @@
         // end of radio button hack
     }
 
+    &__col {
+        &:last-of-type {
+            text-align: right;
+        }
+
+        &:first-of-type {
+            text-align: left;
+        }
+    }
+
+    //To keep the column size consistent between rows in the list.
+    &__cols {
```

Comment:
I think this is forcing the flexbox to act like a grid, it may be a better shout to use grid rather than flexbox if needing to achieve this. This would mean a new component if we are aiming to keep all columns even width across all rows as I think this component wasn't initially designed to be a table but just a simple list (as per the name). Aware this would inflate this task considerably.

From looking at the designs though, I think we can get away with still using this component but it doesn't need to have even columns across the rows. So maybe we can get away with removing this functionality and the colCount logic? It is possible that I have missed a use case though if so happy to discuss it further.

Reply:
It doesn't explicitly say the columns need to be evenly sized on the ticket. I was basing it on the design attached to the ticket which shows the user 'chips' aligned on the left between the rows.
Happy to take this out if we don't think its needed, I wasn't happy with the code anyway.

Reply:
Going to resolve this thread as we have discussed and we are going to continue with this current implementation as the work has been done